examples/wines.get.json

```diff
@@ -282,6 +282,28 @@
                         "score": 1
                     }
                 ],
+                "characteristics": [
```

Comment:
@curzio-della-santa, what do you think about `characteristic_scores` to make it more clear?

Reply:
ok